src/test/java/HttpServerTest.java

```diff
@@ -0,0 +1,104 @@
+import org.apache.http.HttpResponse;
+import org.apache.http.client.fluent.Request;
+import org.junit.*;
```

Comment:
No wildcard import is accepted due to style guide.

Reply:
fixed